### PR TITLE
Bugfix user build

### PR DESCRIPTION
**Pull request by bvanessen**
Fixes the cache issues on Tioga by reverting back to `/tmp` and adds some helping commands to avoid building duplicate spack packages that are externally available.

### CODE REVIEW

File: ci_test/common_python/tools.py [hunk 1]
```diff
@@ -753,16 +753,16 @@ def create_tests(setup_func,
         else:
             environment = {}
 
-        if os.environ.get('USER') == 'lbannusr':
-            basepath = '/p/vast1/lbannusr'
-        else:
-            basepath = '/p/vast1/lbann'
+        # if os.environ.get('USER') == 'lbannusr':
```
Comment (tbennun) [inline ci_test/common_python/tools.py:756]: We should add a comment as to why we commented this out.

**Review by tbennun — CHANGES_REQUESTED** [review]
Minor change

File: ci_test/common_python/tools.py
Comment (benson31) [inline ci_test/common_python/tools.py:1]: Why is there a change in `ci_test` when the PR is to fix the user build?

File: ci_test/common_python/tools.py (hunk 1 above)
Comment (bvanessen) [reply ci_test/common_python/tools.py:756]: I had the comment in there `        # Empirically the cache dir cannot be on a parallel file system`

**Review by benson31 — APPROVED** [review]
LGTM

**Pull request merged**